packages/core/src/browser/style/dialog.css

```diff
@@ -76,7 +76,7 @@
 
 .p-Widget.dialogOverlay .dialogControl {
     padding: calc(var(--theia-ui-padding)*2);
-    padding-top: 0px;
+    padding-top: var(--theia-ui-padding);
```

Comment:
I don't even understand why `padding-top` was set to 0 before, but if we're going to add padding again why not remove that rule completely? It looks better to me this way in our current scenarios.

cc @vince-fugnitto?

Reply:
for me personally it looked better with just the single padding instead of the double. But sure i can change it

Reply:
"When undefined the whole pre-filled value will be selected"